app/components/preprint-form-section.js

```diff
@@ -39,7 +41,7 @@ export default CpPanelComponent.extend(Analytics, {
      */
     hasOpened: false,
 
-    trackOpenState: Ember.observer('isOpen', function() {
+    trackOpenState: observer('isOpen', function() {
```

Comment:
Is there any particular reason why we are still using observers here? This might not be within the scope of this ticket, but it will be nice if we can avoid using observers as much as possible.